src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,7 @@
+package org.example;
```

Comment:
This entire file and folder should be deleted from this repo. It may be causing the build issue. 
Delete org/example - folder
Main.java  - file 

![image](https://github.com/SWE-CS410/5540-Spring23CodeReviewActivity/assets/71608163/79d9f939-93cd-46fa-a78e-24ec8a2d42db)

Reply:
you are right. It is run successful after I delete the org/example